### PR TITLE
devise-jwt seems to be working

### PR DESCRIPTION
**Pull request by leighpoulier**

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -46,6 +46,7 @@ gem 'devise-jwt', '~> 0.9.0'
 
 gem 'aws-sdk-s3', '~> 1.111'
 
-gem 'cors', '~> 1.0'
 
 gem "rspec-rails", "~> 5.1", :groups => [:development, :test]
+
+gem "rack-cors", "~> 1.1"
```
Comment (leighpoulier) [inline Gemfile]: 'rack-cors' seems to be the gem people are using, not 'cors'
https://medium.com/ruby-daily/a-devise-jwt-tutorial-for-authenticating-users-in-ruby-on-rails-ca214898318e
https://tergell.medium.com/rails-devise-jwt-tutorial-b5d5b03d9040

File: config/initializers/cors.rb [hunk 2]
```diff
@@ -1,17 +1,24 @@
...
+    else
+      origins '*'
+      puts "cors origin: *"
+    end
+
+    resource '*',
+      headers: :any,
+      methods: [:get, :post, :put, :patch, :delete, :options, :head]
+  end
+end
```
Comment (leighpoulier) [inline config/initializers/cors.rb:24]: for CORS, attempted to allow all origins in 'test' and 'development' environments, but restrict to 'mexiqui.to' for production.

File: config/initializers/session_store.rb [hunk 3]
```diff
@@ -0,0 +1,2 @@
+# Session store disabled as per devise-jwt setup at https://github.com/waiting-for-dev/devise-jwt
+Rails.application.config.session_store :disabled
```
Comment (leighpoulier) [inline config/initializers/session_store.rb]: this is part of the devise-jwt instructions, and it says if you're using API only rails, then you most likely don't need sessions, so disable them.
https://github.com/waiting-for-dev/devise-jwt

File: Gemfile (hunk 1 above)
Comment (koray-eren) [reply Gemfile]: yeah i saw that yesterday when i was googling something related

**Review by koray-eren — APPROVED** [review]

**Pull request merged**